presto-hive/src/test/java/io/prestosql/plugin/hive/s3/MockAmazonS3.java

```diff
@@ -96,25 +96,25 @@ public PutObjectResult putObject(PutObjectRequest putObjectRequest)
     }
 
     @Override
-    public ObjectListing listObjects(ListObjectsRequest listObjectsRequest)
+    public ListObjectsV2Result listObjectsV2(ListObjectsV2Request listObjectsV2Request)
     {
-        ObjectListing listing = new ObjectListing();
+        ListObjectsV2Result listingV2 = new ListObjectsV2Result();
 
         S3ObjectSummary standard = new S3ObjectSummary();
         standard.setStorageClass(StorageClass.Standard.toString());
         standard.setKey("test/standard");
         standard.setLastModified(new Date());
-        listing.getObjectSummaries().add(standard);
+        listingV2.getObjectSummaries().add(standard);
```

Comment:
Could we also have test that uses `setContinuationToken` path?

Reply:
Added. But I'm not sure if it is a right way. Please help to review, thanks~